subsys/logging/Kconfig.mode

```diff
@@ -36,14 +36,21 @@ config LOG_MODE_MINIMAL
 	  colors, or asynchronous logging, and all messages are simply
 	  sent to printk().
 
+endchoice
+
 config LOG_FRONTEND
 	bool "Frontend"
 	help
-	  When enabled, logs are redirected to a custom frontend instead
-	  of being processed by the logger. In this mode runtime filtering and
-	  multiple backends are not used.
+	  When enabled, logs are redirected to a custom frontend which is the
+	  fastest way of getting logs out.
 
-endchoice
+config LOG_FRONTEND_ONLY
+	bool "No backends"
```

Comment:
Maybe `bool "No backends" if !LOG1`? Since v1 cannot have both frontend and backends enabled, removing the prompt prevents this from being disabled.